extension/src/popup/components/sendPayment/SendAmount/index.tsx

```diff
@@ -224,13 +254,19 @@ export const SendAmount = ({ previous }: { previous: ROUTES }) => {
         >
           <>
             <input
```

Comment:
let's add `autocomplete="off"` to this field to keep it from suggesting values you've put in in the past

Reply:
ooh good call https://github.com/stellar/freighter/pull/425/commits/d4d9a00a59c2fc45fae60d93547194e717826030